### DISCUSSION TITLE
How to remove el-dialog__body padding?

### DISCUSSION THREAD

**Discussion by premdasvm** [element-plus/element-plus#6639] [Q&A]
Hi, I've a form wrapped in `el-dialog`, when I inspect the page I can see a div with class `el-dialog__body` how can I remove the top padding of this?

**Comment by msidolphin** [comment]
We need to add a css var for this, you can temporarily override the padding

**Reply by premdasvm** [reply]
How do I do that? Can u share an example repo or code ? I'm not that used to css var or SCSS.

**Reply by jw-foss** [reply]
Generally you can do something like this: 

<img width="1203" alt="image" src="https://user-images.githubusercontent.com/15975785/158538155-a1699818-3710-4e76-8521-5ce887e524e5.png">

You can make your own `index.[s]css` to include code: 

```css
.your-class-name {
  --el-dialog-padding-primary: -10px;
}
```

```html
<el-dialog custom-class="your-class-name" />
```

The whole idea here is that you pass the class name as `custom-class` to `dialog` then you include a `cssvar` of your own to override the built-in token.

**Reply by jw-foss** [reply]
Check [this](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtYnV0dG9uIHR5cGU9XCJ0ZXh0XCIgQGNsaWNrPVwiY2VudGVyRGlhbG9nVmlzaWJsZSA9IHRydWVcIlxuICAgID5DbGljayB0byBvcGVuIHRoZSBEaWFsb2c8L2VsLWJ1dHRvblxuICA+XG5cbiAgPGVsLWRpYWxvZyB2LW1vZGVsPVwiY2VudGVyRGlhbG9nVmlzaWJsZVwiIHRpdGxlPVwiV2FybmluZ1wiIHdpZHRoPVwiMzAlXCIgY3VzdG9tLWNsYXNzPVwibXktY2xhc3NcIj5cbiAgICA8c3BhblxuICAgICAgPkl0IHNob3VsZCBiZSBub3RlZCB0aGF0IHRoZSBjb250ZW50IHdpbGwgbm90IGJlIGFsaWduZWQgaW4gY2VudGVyIGJ5XG4gICAgICBkZWZhdWx0PC9zcGFuXG4gICAgPlxuICAgIDx0ZW1wbGF0ZSAjZm9vdGVyPlxuICAgICAgPHNwYW4gY2xhc3M9XCJkaWFsb2ctZm9vdGVyXCI+XG4gICAgICAgIDxlbC1idXR0b24gQGNsaWNrPVwiY2VudGVyRGlhbG9nVmlzaWJsZSA9IGZhbHNlXCI+Q2FuY2VsPC9lbC1idXR0b24+XG4gICAgICAgIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIiBAY2xpY2s9XCJjZW50ZXJEaWFsb2dWaXNpYmxlID0gZmFsc2VcIlxuICAgICAgICAgID5Db25maXJtPC9lbC1idXR0b25cbiAgICAgICAgPlxuICAgICAgPC9zcGFuPlxuICAgIDwvdGVtcGxhdGU+XG4gIDwvZWwtZGlhbG9nPlxuPC90ZW1wbGF0ZT5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnXG5cbmNvbnN0IGNlbnRlckRpYWxvZ1Zpc2libGUgPSByZWYoZmFsc2UpXG48L3NjcmlwdD5cbjxzdHlsZT5cbi5kaWFsb2ctZm9vdGVyIGJ1dHRvbjpmaXJzdC1jaGlsZCB7XG4gIG1hcmdpbi1yaWdodDogMTBweDtcbn1cbiAgXG4uZWwtZGlhbG9nLm15LWNsYXNzIHtcbiAgLS1lbC1kaWFsb2ctcGFkZGluZy1wcmltYXJ5OiAtMTBweDtcbn1cbjwvc3R5bGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XCJpbXBvcnRzXCI6e319In0=) out

**Note that** if you changed this variable, it should be reflected on both the header and the body
